### ISSUE TITLE
How to unzip a file of images into a list and extract its metadata (EXIF files)

### ISSUE THREAD

**Issue by daen1on** [Stuk/jszip#484]
Hi, I'm new at programming in Javascript and I'm having trouble at understanding how to use your library correctly, I can't get how to unzip a file of images and show them in a list. I don't how to take every image and separe them for take its EXIF files. Help please, My main trouble is that i can't extract the images. Its a little bit hard for me to understand the JSZip library. :(

**Comment by Sasamq** [comment]
#484

**Comment by surajsharma** [comment]
yes i would like an answer to this question as well since i am in the same boat as daen1on
